circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/NavEvent.kt

```diff
@@ -34,9 +30,19 @@ public sealed interface NavEvent : CircuitUiEvent {
   public data class GoTo(val screen: Screen) : NavEvent
 
   /** Corresponds to [Navigator.resetRoot]. */
-  public data class ResetRoot(
-    val newRoot: Screen,
-    val saveState: Boolean,
-    val restoreState: Boolean,
-  ) : NavEvent
+  public data class ResetRoot(val newRoot: Screen, val options: StateOptions) : NavEvent {
+    @Deprecated(
```

Comment:
The Boolean override is a nice convenience for the common case, anything against keeping that?